src/emu/cheat.h

```diff
@@ -294,7 +295,7 @@ class cheat_manager
 	// getters
 	running_machine &machine() const { return m_machine; }
 	bool enabled() const { return !m_disabled; }
-	cheat_entry *first() const { return m_cheatlist.first(); }
+	const auto &entries() const { return m_cheatlist; }
```

Comment:
not sure this will work on GCC 4.9 which is minimal requirement, and highest available on some platforms.